### PR TITLE
Fix for OSD grid overflowing the preview area as per #3202

### PR DESCRIPTION
**Pull request by benlumley**
The overflowing issue in the screenshots in #3202 is caused by the images for each 'character' not shrinking below 12px wide; this css allows them to do that and then prevents the overflow.

### CODE REVIEW

**Comment by McGiverGim** [comment]
Can you post some screenshots of images under the normal size? I'm out of the computer to test it by myself. I think remember when I tried it that the result was bad.

**Comment by benlumley** [comment]
> Can you post some screenshots of images under the normal size? I'm out of the computer to test it by myself. I think remember when I tried it that the result was bad.

Will do a little later on.

In testing, it was a bit sketchy; gets v.small; the icons in particular are hard to make out. This isn't really avoidable though; at least not without a bigger rework of the whole OSD tab? And it's less bad than overflowing I think...

Anyway - I'll get some screenshots sure!

**Comment by McGiverGim** [comment]
Earlier versions simply limited the minimum width to the size needed to fit the complete row without reduction.

**Comment by benlumley** [comment]
Also - here's without the fix, replicating the issue from #3202 

OSD way more legible due to bigger characters; but ultimately it doesn't fit. 

Screenshot 2023-01-03 at 13 09 59

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
@benlumley have tried to fix the viewport for Android but it's a mess as you say, works on some phones, others not, my latest Nokia has the preview floated right somehow. Maybe you have a better solution?

**Review by blckmn — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Comment by benlumley** [comment]
> @benlumley have tried to fix the viewport for Android but it's a mess as you say, works on some phones, others not, my latest Nokia has the preview floated right somehow. Maybe you have a better solution? I had to make the preview bigger for OSD so with your change this might be reverted or make dynamic depending on canvas size?

I'll do a bit of testing and see!

**Pull request merged**
